src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -104,8 +105,12 @@ export default class DefaultAudioVideoController
   private enableSimulcast: boolean = false;
   private totalRetryCount = 0;
   private startAudioVideoTimestamp: number = 0;
+  private signalingTask: Task;
   destroyed = false;
 
+  /** @internal */
+  usePromises: boolean = true;
+
```

Comment:
Leaving this in for a little while to have an easier way to revert this change if needed.